WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -32,34 +52,28 @@ class GutenbergLayoutPickerViewController: UIViewController {
         }
     }
 
-    var completion: PageCoordinator.TemplateSelectionCompletion? = nil
-
     private var shouldUseCompactLayout: Bool {
         return traitCollection.verticalSizeClass == .compact
     }
 
+    private var _maxHeaderHeight: CGFloat = 0
     private var maxHeaderHeight: CGFloat {
         if shouldUseCompactLayout {
             return minHeaderHeight
         } else {
-            return largeTitleView.frame.height +
-            midHeaderHeight
```

Comment:
Switched these `(max|mid|min)HeaderHeight` to be stored values, because once I started loading the table view with data I found inconsistency with how the header and footers would layout depending on layout timing.  Controlling this value just gave me a better experience.